méli-mélo/2022-09-svgimagemap/styles.css

```
@@ -0,0 +1,60 @@
+.mapdiv {
+	width: 33%;
+	margin: auto;
+}
+
+.mapdiv path {
+	stroke: #FFFFFF;
+	stroke-width: 5px;
+}
+
+.mapdiv polygon {
+	stroke: #FFFFFF;
+	stroke-width: 5px;
```

```suggestion
	stroke-width: 14px;
```

to be rendered as 2 px when the SVG are resized from 2544px to 376px